src/fn/compile-app-settings.js

```diff
@@ -39,7 +39,30 @@ const compileAppSettingsForProject = async (projectDir, options) => {
   }
 
   const readOptionalJson = path => fs.exists(path) ? fs.readJson(path) : undefined;
-  const appSettings = fs.readJson(path.join(projectDir, 'app_settings.json'));
+  // Manual configurations should be done in the base-settings.json file
+  // This check and warning can be removed when all project configurations have this new file defined
+  let appSettings = fs.readJson(path.join(projectDir, 'app_settings.json'));
```

Comment:
This line will thrown an error if the app_settings file does not exist. Does this mean we expect configurers to create this file and leave it empty if they wish to go the modular route?

Reply:
Added this so as not to break current configurations. Will refactor based on the suggestion below on checking if config is modular.